firefox-ios/firefox-ios-tests/Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -5,15 +5,58 @@
 import XCTest
 import Common
 
-// swiftlint:disable line_length
-let standardBlockedElementsString = "Firefox blocks cross-site trackers, social trackers, cryptominers, and fingerprinters."
-let strictBlockedElementsString = "Firefox blocks cross-site trackers, social trackers, cryptominers, fingerprinters, and tracking content."
-// swiftlint:enable line_length
-
+// Urls
 let websiteWithBlockedElements = "twitter.com"
 let differentWebsite = path(forTestPage: "test-example.html")
+let trackingProtectionTestUrl = "https://senglehardt.com/test/trackingprotection/test_pages/"
```

Comment:
How stable is this website? We have had tests that depends on external sites that fail occasionally because the site was slow or implements a CAPTCHA.

Reply:
You're right @clarmso . So far the page is pretty stable but once we use it in a CI environment maybe we can have some issues. The page is a very simple page, with just some links and the test is only opening the webpage, is not using any content. The manual test case use this url, this is why I used the same url. The point is that to check the tracking protection we have to check an element in the padlock in the urlbar, so technically we can use any page.

Reply:
The webpage is a mozilla webpage so it should safe to use